### ISSUE TITLE
Image alt text not being converted to alt tags in PDF/UA

### ISSUE THREAD

**Issue by brendan-ward** [Kozea/WeasyPrint#1872]
Thank you for this library, it has been very helpful for our work!

We are creating PDFs that contain a lot of map images using `variant="pdf/ua-1"`.

Within our HTML, we are assigning these `alt` tags in the normal way for HTML.

```
<img src="..." alt="My image" />
```

However, these seem to be missing Alt tags when converted to PDF/UA, and so they are failing accessibility checks.

I don't quite follow how these are converted into tags in PDF/UA, but it seems like it should be possible in theory to retain the alt tag when processing the image and convert it into the associated alt tag in PDF/UA.

I didn't see examples to clarify if there was another way to properly tag images.

Possibly related: #1838

**Comment by liZe** [comment]
Hi!

Yes, unfortunately, many things are not handled with PDF/UA support yet.

> I don't quite follow how these are converted into tags in PDF/UA, but it seems like it should be possible in theory to retain the alt tag when processing the image and convert it into the associated alt tag in PDF/UA.

Tags are set in [the `stream.py` file](https://github.com/Kozea/WeasyPrint/blob/0eb4e14cd3df5264a101dc04bda074dbd05208dc/weasyprint/pdf/stream.py#L458) and structured in [the `pdfua.py` file](https://github.com/Kozea/WeasyPrint/blob/master/weasyprint/pdf/pdfua.py), where you have ElementTree elements (ie. HTML tags transformed into Python objects) and thus `alt` attributes. By changing these two functions, it should be possible to put the `alt` attributes into correct PDF tags.

**Comment by Tomefa** [comment]
+1 interested also for the "alt" on image during PDF export

**Comment by liZe** [comment]
> +1 interested also for the "alt" on image during PDF export

If someone wants to add the feature, I’ll be happy to help! The previous comment already contains useful tips.

**Comment by tsmcgrath** [comment]
Another +1 from me. I think it may be beyond my coding abilities to help add it though. Seeing if I can get some help to add this feature. It's for a non-profit project so I wouldn't be able to make much of a donation to cover the work. I'd consider some donation though. Say $100 USD. Let's chat if interested. I'm trying to recruit another serious python engineer to help too.

**Comment by liZe** [comment]
If you’re interested in this issue, it’s time to test #2471! Feedback will be highly appreciated, even to say that it just works. 🙏
